src/modules/events/send_event.h

```diff
@@ -51,46 +51,93 @@ extern "C" __EXPORT int send_event_main(int argc, char *argv[]);
+
+	/**
+	 * @brief Default Constructor.
+	 */
 	SendEvent();
-	~SendEvent();
 
 	/**
-	 * Initialize class in the same context as the work queue. And start the background listener.
-	 * @return 0 if successful, <0 on error */
-	static int task_spawn(int argc, char *argv[]);
+	 * @brief Default Destructor.
+	 */
+	~SendEvent();
 
-	/** @see ModuleBase */
+	/**
+	 * @brief Recognizes custom commands.
+	 * @param argc The task argument count.
+	 * @param argc Pointer to the task argument variable array.
+	 * @see ModuleBase for base class inheritance.
+	 * @return Returns 0 iff successful, otherwise < 0 on error.
+	 */
 	static int custom_command(int argc, char *argv[]);
 
-	/** @see ModuleBase */
+	/**
+	 * @brief Prints usage options to the console.
+	 * @param reason Pointer to... (something important? Perhaps the number 42?... :) )
+	 *               @todo: Assitance required to fill in intent for the param 'reason'.
+	 * @see ModuleBase for base class inheritance.
+	 * @return Returns 0 iff successful, otherwise < 0 on error.
+	 */
 	static int print_usage(const char *reason = nullptr);
 
-private:
+	/**
+	 * @brief Initialize class in the same context as the work queue. And start the background listener.
+	 * @param argc The task argument count.
+	 * @param argc Pointer to the task argument variable array.
+	 * @return Returns 0 iff successful, otherwise < 0 on error.
+	 */
+	static int task_spawn(int argc, char *argv[]);
 
-	/** Start background listening for commands
-	 *
-	 * @return 0 if successful, <0 on error. */
-	int start();
+private:
 
+	/**
+	 * @brief Returns an ACK to a vehicle_command.
+	 * @param cmd The vehicle command struct referenced.
+	 * @param result The command acknowledgement result.
+	 */
+	void answer_command(const vehicle_command_s &cmd, unsigned result);
 
-	/** Trampoline for initialisation. */
-	static void initialize_trampoline(void *arg);
-	/** Trampoline for the work queue. */
+	/**
+	 * @brief Trampoline for the work queue.
+	 * @param arg Pointer to the task startup arguments.
+	 */
 	static void cycle_trampoline(void *arg);
 
-	/** call process_commands() and schedule the next cycle. */
+	/**
+	 * @brief Calls process_commands() and schedules the next cycle.
+	 */
 	void cycle();
 
-	/** check for new commands and process them. */
+	/**
+	 * @brief Trampoline for initialisation.
+	 * @param arg Pointer to the task startup arguments.
+	 */
+	static void initialize_trampoline(void *arg);
+
+	/**
+	 * @brief Checks for new commands and processes them.
+	 */
 	void process_commands();
 
-	/** return an ACK to a vehicle_command */
-	void answer_command(const vehicle_command_s &cmd, unsigned result);
+	/**
+	 * @brief Starts background task listening for commands.
+	 * @return Returns 0 iff successful, otherwise < 0 on error.
+	 */
+	int start();
 
+	/** @struct _work */
 	static struct work_s _work;
+
+	/** @param _subscriber_handler */
 	SubscriberHandler _subscriber_handler;
+
+	/** @param _status_display */
```

Comment:
Again: such a comment does not add value.
Also can you use the form that we already use: `status::StatusDisplay *_status_display = nullptr;   /**< comment */`

Reply:
The only reason this one doesn't add value is because I don't understand it yet.  Thank you for pointing it out!

Reply:
I think that this param could be documented such that it's intent is easily understood! :)

Reply:
Hi @bkueng , I didn't do a good job of answering your question:
> Also can you use the form that we already use: status::StatusDisplay *_status_display = nullptr; /**< comment */

I couldn't see that this was a convention within this file. there were just no other instances to indicate.  Is this the preferred convention?  This is somewhat different than I've observed in other code bases, but if it's the style here, I'll modify the PR!  Thanks again for your feedback!

-Mark

Reply:
> I couldn't see that this was a convention within this file.

Not within this file, but in general throughout the code base (it might not be consistent everywhere but it's definitely the majority).